mysql-test/suite/sys_vars/t/innodb_buffer_pool_dump_pct_function.opt

```diff
@@ -0,0 +1,5 @@
+--loose-default-storage-engine=innodb
+--loose-innodb_buffer_pool_load_at_startup=0
+--loose-innodb_buffer_pool_dump_at_shutdown=0
+--loose-innodb-buffer-pool-size=8M
+--loose-innodb-page-size=16k
```

Comment:
Good, we want to test with a specific page size, so that the number of BLOB pages is constant.